pages/common/loc.md

```diff
@@ -0,0 +1,19 @@
+# loc
+
+> Tool written in Rust that counts lines of code quickly.
+
+- Print lines of code in the current directory:
+
+`loc`
+
+- Print lines of code in the target directory:
+
+`loc {{target}}`
+
+- Print lines of code with stats for individual files:
+
+`loc --files`
+
+- Print lines of code without .gitignore (etc.) files. Two -u flags will additionally count hidden files and dirs:
```

Comment:
The second sentence can be in parenthesis.

Reply:
Probably better format then. Thanks.